sdk-tests/src/test/java/io/dapr/it/DaprPorts.java

```diff
@@ -30,10 +34,18 @@ public class DaprPorts {
 
   private final Integer appPort;
 
+  private final Map<Property<?>, String> overrides;
+
   private DaprPorts(Integer appPort, Integer httpPort, Integer grpcPort) {
     this.grpcPort = grpcPort;
     this.httpPort = httpPort;
     this.appPort = appPort;

```

Comment:
We are using Java 11, so there is no need to use old Java patterns like `Collections.unmodifiableMap(...)` you could replace all that with:
```
this.overrides = Map.of(
      Properties.GRPC_PORT, grpcPort.toString(),
      Properties.HTTP_PORT, httpPort.toString(),
      Properties.HTTP_ENDPOINT, "http://127.0.0.1:" + httpPort,
      Properties.GRPC_ENDPOINT, "127.0.0.1:" + grpcPort
    );
```
Which I think is more readable and `Map.of()` creates an immutable map behind the scenes.